src/slur.cpp

```diff
@@ -364,18 +363,18 @@ void Slur::AdjustSlurPosition(Doc *doc, FloatingCurvePositioner *curve,
         }
         if (dist != 0) posXRatio = (float)posX / ((float)dist / 2.0);
 
-        shift = 0;
         // Keep the maximum shift on the left and right
+        int shift;
         if (curveDir == curvature_CURVEDIR_above) {
             shift = intersection;
         }
         else {
-            shift = intersection;
+            shift = -intersection;
```

Comment:
Why are you changing this?

Reply:
I don't understand logic of calculation. 
But doing same in both branches of `if` suggests here is a bug. 
I thought it was meant something like that.
No? What should it be then?

Reply:
OK. I have to look at this because we need to make sure we fix it in the right direction.

Reply:
@lpugin what's with this PR?

Reply:
Nothing from my side. Current calculation is definitely wrong and has a bug. But I am unable to fix it.

Reply:
I can see that doing then same in a if/else makes no sense, but could provide us with an MEI snippet that illustrates the bug?

Reply:
No, I can't provide MEI. Having such MEI I would probably have hints how to fix.